### PR TITLE
Token-Allocation EA should support blocksize-capital EA

### PR DESCRIPTION
**Pull request by cawthorne**
## Description

Add support for the blocksize-capital EA in the token-allocation EA 

## Changes

Added a new BLOCKSIZE_CAPITAL_ADAPTER_URL and new blocksize-capital sources option.

## Steps to Test

yarn test token-allocation

### CODE REVIEW

File: packages/non-deployable/token-allocation/README.md [hunk 1]
```diff
@@ -8,21 +8,22 @@ The adapter takes the following environment variables:
...
+|           |   `COINMETRICS_ADAPTER_URL  `   |   The location of a CoinMetrics external adapter    |         |             |
+|           |    `COINPAPRIKA_ADAPTER_URL`    |   The location of a CoinPaprika external adapter    |         |             |
+|           |    `COINRANKING_ADAPTER_URL`    |   The location of a CoinRanking external adapter    |         |             |
+|           |   `CRYPTOCOMPARE_ADAPTER_URL`   |  The location of a CryptoCompare external adapter   |         |             |
+|           |      `FINAGE_ADAPTER_URL`       |      The location of a Finage external adapter      |         |             |
+|           |       `KAIKO_ADAPTER_URL`       |      The location of a Kaiko external adapter       |         |             |
+|           |       `NCFX_ADAPTER_URL`        |       The location of a NCFX external adapter       |         |             |
+|           |      `TIINGO_ADAPTER_URL`       |      The location of a Tiingo external adapter      |         |             |
+|           | `BLOCKSIZE_CAPITAL_ADAPTER_URL` | The location of a BlocksizeCapital external adapter |         |             |
```
Comment (austinborn) [inline packages/non-deployable/token-allocation/README.md:26]: Small nit, would you be able to alphabetize the new lines in the README?

**Review by austinborn — DISMISSED** [review]
Great! Might ask if you can keep the new lines in alphabetized order (at least in the README), but otherwise looks good 👍

**Comment by cawthorne** [comment]
> Great! Might ask if you can keep the new lines in alphabetized order (at least in the README), but otherwise looks good 👍

Hi, I have alphabetized the README :)

File: packages/non-deployable/token-allocation/README.md (hunk 1 above)
Comment (cawthorne) [reply packages/non-deployable/token-allocation/README.md]: Done!

File: packages/non-deployable/token-allocation/README.md [hunk 2]
```diff
@@ -38,12 +39,12 @@ See the [Composite Adapter README](../README.md) for more information on how to
...
+| Required? |     Name      |                                    Description                                     |                                                                                           Options                                                                                            |                Defaults to                |
+| :-------: | :-----------: | :--------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :---------------------------------------: |
+|           |   `source`    | The data provider to query data from. This is required if not specified in config. | `amberdata`, `cfbenchmarks`, `coinapi`, `coingecko`, `coinmarketcap`, `coinmetrics`, `coinpaprika`, `coinranking`, `cryptocompare`, `finage`, `kaiko`, `ncfx`, `tiingo`, `blocksize_capital` |                                           |
```
Comment (austinborn) [inline packages/non-deployable/token-allocation/README.md:44]:
```suggestion
|           |   `source`    | The data provider to query data from. This is required if not specified in config. | `amberdata`, `blocksize_capital`, `cfbenchmarks`, `coinapi`, `coingecko`, `coinmarketcap`, `coinmetrics`, `coinpaprika`, `coinranking`, `cryptocompare`, `finage`, `kaiko`, `ncfx`, `tiingo` |                                           |
```

**Review by austinborn — APPROVED** [review]
Looks good, thank you!

**Pull request merged**